packages/shared/styles/components/molecules/SfRadio.scss

```diff
@@ -39,6 +40,11 @@
     position: absolute;
     opacity: 0;
   }
+  input {
+    &:focus + .sf-radio__container {
+      --radio-checkmark-box-shadow: 0 0 2px 2px rgb(59, 153, 252);
```

Comment:
Nice solution @martaradziszewska! Clicking space works and it changes on checked. I found one issue - our focus directive doesn't work with box shadow so after mouse click the shadow is still on the element :)

Reply:
@justyna-13, @martaradziszewska I added it to my to-do list and I will try to improve these issues. But now important for as is showing all outlines :)